server/xpub/entities/manuscript/helpers/empty.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * this should be kept in sync with the schema
+ */
+const emptyManuscript = {
+  meta: {
+    title: '',
+    articleType: '',
+    subjects: [],
+  },
+  opposedSeniorEditorsReason: '',
+  opposedReviewingEditorsReason: '',
+  opposedReviewers: [],
+  suggestionsConflict: false,
+  files: [],
+  coverLetter: `
+<p><strong>How will your work make others in the field think differently and move the field forward?</strong></p>
```

Comment:
Should this be kept in a file or a different place in the code?

Reply:
That's a great point, and I agree - especially if we use @damnhipster 's suggestion of using a class it could go on there. However, I think as long as the place is consistent for now we can move this later. Good work @tamlyn  - keen to get this in - maybe we can talk about these refinements f2f on Thursday?